hudi/src/main/scala/org/apache/spark/sql/delta/hudi/HudiSchemaUtils.scala

```diff
@@ -33,16 +33,18 @@ object HudiSchemaUtils extends DeltaLogging {
      * Recursively (i.e. for all nested elements) transforms the delta DataType `elem` into its
      * corresponding Avro type.
      */
-    def transform[E <: DataType](elem: E, isNullable: Boolean): Schema = elem match {
+    def transform[E <: DataType](elem: E, isNullable: Boolean, currentPath: String): Schema =
+      elem match {
       case StructType(fields) =>
 
         val avroFields: util.List[Schema.Field] = fields.map(f =>
           new Schema.Field(
             f.name,
-            transform(f.dataType, f.nullable),
+            transform(f.dataType, f.nullable,
```

Comment:
What is the impact of this change to the reader side?

Reply:
No change other than the functionality works - their tables should look the same as if they created it in Hudi